projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -86,7 +86,10 @@ public function reregister_menu_items() {
 			$this->add_new_site_link();
 		}
 
-		$this->add_woocommerce_installation_menu();
+		if ( 'wp-admin' !== get_option( 'wpcom_admin_interface' ) ) {
```

Comment:
Not sure if we're already starting to use `use_wp_admin_interface` as mentioned here (p1706736745914429-slack-C06DN6QQVAQ)

Reply:
Will switch to that if it's available and merge, thanks for the heads up!